pyiron_atomistics/atomistics/job/interactive.py

```diff
@@ -118,7 +118,6 @@ def run_if_interactive(self):
             self.interactive_initialize_interface()
         if self._structure_previous is None:
             self._structure_previous = self.structure.copy()
-        self._update_previous_structure()
```

Comment:
I do not understand this part - if you move `update_previous_structure()` to line 136 - then in line 123 the structure is not yet updated.

Reply:
Previous structure has to be set to the current structure only after the variable comparisons are done, i.e. after the run.

Reply:
Ok, I see.